src/styles/components/_component-card.scss

```diff
@@ -21,18 +21,50 @@
       margin: auto;
     }
 
+    // the following breakpoints are where certain illustrations try to break out of their containers
+    @media only screen and (min-width: 475px) {
+      height: 30rem; // 480px/16px
+    }
+
+    @media only screen and (min-width: 690px) {
+      height: 40rem; // 640px/16px
+    }
+
+    @media only screen and (min-width: $breakpoint-tablet) {
+      height: 100%;
+    }
+
     &:hover img {
       scale: 102%;
     }
 
     &__header {
-      font-weight: 700;
-      font-size: 1.25rem; // 20px
+      font-family: "Oxygen Bold", sans-serif;
+      z-index: 1;
+      font-size: 1.125rem; // 18px/16
       line-height: 4.6667rem; // 75px
       letter-spacing: 2%;
```

Comment:
Percentages[ can't be used](https://w3c.github.io/csswg-drafts/css-text/#letter-spacing-property) for letter-spacing. You should be able to use rems, but I'm not sure where the 2% comes from so I would want to understand that to determine the correct value.

Reply:
I initially got the 2% from Figma, and didn't remember that percents don't work as a unit. The code I got from Figma says 0.02em for the letter-spacing, so that's what this commit has in it, as well as some other padding/margin fixes. 

https://github.com/sparkbox/accessible-components/pull/58/commits/a2bcc8d42fb465cb190ba70fcfd0337a7cd3928d